src/common/containers/builds.js

```diff
@@ -25,7 +36,7 @@ class Builds extends Component {
         <h1>{fullName} builds</h1>
         <BuildHistory account={account} repo={repo}/>
         { this.props.isFetching &&
-          <span>Loading...</span>
+          <div className={styles.spinner}><Spinner /></div>
```

Comment:
Yeah, I think repeatedly popping up the spinner will get pretty distracting.  I'd stick to devtools for demonstrating that auto-refresh works.